pocketimdb/movies/views.py

```diff
@@ -48,3 +48,18 @@ def increment_visits(self, request, pk):
         movie.visits = movie.visits + 1
         movie.save()
         return Response(status=HTTP_200_OK)
+
+    @action(methods=['POST'], detail=True, url_path='post-comment')
```

Comment:
ruta za komentare bi trebala da ide `movies/<:id>/comments` i onda nadovezujes za GET, POST, PUT, DELET po REST princpiu

Reply:
Da bih ovako nesto uradio, mislim da moram onda koristiti onu dodatnu biblioteku za nested routes.

Reply:
Tj ne mora pomocu toga, vec moze na onaj nacin kada sam za likes rutu radio, samo si bio rekao da to bas i ne valja. Na ovo sa slike mislim.

<img width="1102" alt="Screen Shot 2021-01-25 at 14 17 46" src="https://user-images.githubusercontent.com/44477868/105711005-37813300-5f18-11eb-989b-0b51d9816867.png">

Reply:
Na taj nacin bih mogao da imam comments putanju i na nju da vezem razlicite metode. Ili da probam sa bibliotekom za nested routes, samo mi tu koliko sam shvatio treba zasebni View ili ViewSet za tu podputanju, sto ja u ovom slucaju nemam.

Reply:
mozes probati sa onom biblitokeom, prije ili kasnije ces se susresti, tako da probaj nju da iskoristis, mislim da sam na nekom PR-u objasnio na koji nacin da implemntiras pa pogledaj